requirements.yml

```diff
@@ -47,7 +47,7 @@
 
 - name: load-balancer
   src: https://github.com/open-craft/ansible-load-balancer
-  version: b729113c24d877aae44246ed4888eb73f773bd19
+  version: 52ae02307d18b7d693113d35523cb4ed045d3ff0
```

Comment:
Cool, I've done that with d1ea6d82f3b1eb16ca152b996e6c3bf6759b0d4d for all our requirements forks which are currently in line with `origin/master`.

Note: there are two remaining that match commits on `jbzdak`'s branches, so I left them on those branches.

Could you please check my work?